backend/src/main/java/com/redhat/cloud/notifications/auth/principal/rhid/RhServiceAccountIdentity.java

```diff
@@ -0,0 +1,41 @@
+package com.redhat.cloud.notifications.auth.principal.rhid;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class RhServiceAccountIdentity extends RhIdentity {
+
+    @JsonProperty("org_id")
+    private String orgId;
+
+    @JsonProperty("service_account")
+    private ServiceAccount serviceAccount;
+
+    public ServiceAccount getServiceAccount() {
+        return serviceAccount;
+    }
+
+    @Override
+    public String getOrgId() {
+        return orgId;
+    }
+
+    @JsonIgnoreProperties(ignoreUnknown = true)
+    public static class ServiceAccount {
+        @JsonProperty("username")
+        private String username;
+
+        @JsonProperty("client_id")
+        private String clientId;
+
+        public String getUsername() {
+            return username;
+        }
+    }
+
+    @Override
+    public String getName() {
+        return getServiceAccount().getUsername();
```

Comment:
The `username` of a service account looks like metadata to me, not something we'd use in the `ConsolePrincipal`. Should this return `getServiceAccount().getClientId()` instead?